common/OpTestHMC.py

```diff
@@ -285,12 +285,12 @@ def migrate_lpar(self, src_mg_system=None, dest_mg_system=None, options=None, pa
             raise OpTestError("Source and Destination Managed System required for LPM")
         if not self.is_lpar_in_managed_system(src_mg_system, self.lpar_name):
             raise OpTestError("Lpar %s not found in managed system %s" % (self.lpar_name, src_mg_system))
-        self.ssh.run_command(
-            'migrlpar -o v -m %s -t %s -p %s' % (src_mg_system, dest_mg_system, self.lpar_name))
-        cmd = 'migrlpar -o m -m %s -t %s -p %s %s' % (src_mg_system, dest_mg_system, self.lpar_name, param)
-        if options:
```

Comment:
please include this changelog also in the commit message or commit message should be more elaborative

Reply:
this is more of code optimization to reduce lines

Reply:
that review comment for entire code change as I see commit message where these changes apply not convey the message properly for an external reviewer